packages/odyssey/CHANGELOG.md

```diff
@@ -5,6 +5,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.0] - 2020-07-31
+
+### Added
+
+- `@okta/odyssey-icons` package: includes an initial set of SVGs and process for batch optimizing via `svgo`
+- `@okta/vuepress-theme-nimatron` package: supplies the default theme for Odyssey's docs
+
+### Changed
+
+- [Breaking Change] Changed the Odyssey documentation platform from `hexo` to `vuepress`
+- `:disabled` and `:read-only` labels now share text and border styling
+- Tab indicators no longer animate via CSS vars
+
+### Removed
+
+- [Breaking Change] `$border-default` token has been removed
```

Comment:
It's common for library authors to provide migration steps for breaking changes like this.

```suggestion
- [Breaking Change] `$border-default` token has been removed. Utilize `$border-color-display` and `$border-color-ui` instead.
```

Reply:
`$border-color-display` and `$border-color-ui` were replacements for `$base-border-color` - this is a previously deprecated variable we left hanging around for a bit too long with no replacement solution.

Reply:
Interesting. Not sure what is correct - as the issue described in #579 states that it was _deprecated_ by those two styles. @arnoldsandoval-okta - can you confirm?

Reply:
Sorry for the confusion, shred is correct. They don't do the same thing, `border-default` provided the shorthand for styling the border color, width, and style. `$base-border-color` was previously deprecated when `$border-color-ui` and `$border-color-display` were introduced.

 Will update #579 for posterity.